server/models/user.js

```diff
@@ -1,3 +1,14 @@
+
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const userSchema = new Schema({
+  id: mongoose.Schema.Types.ObjectId,
+  name: { type: String, required: true },
+  email: { type: String, required: true, unique: true },
+  password: { type: String, required: true },
+});
+
+const User = mongoose.model("User", userSchema);
```

Comment:
Make sure to periodically get the latest `dev` branch and merge it into yours to keep it up to date, there are lots of properties missing from this userSchema right now